test/GitcoinGovernor.t.sol

```diff
@@ -441,16 +469,6 @@ contract NewGitcoinGovernorProposalTest is GitcoinGovernorProposalTestHelper {
   uint8 constant FOR = 1;
   uint8 constant ABSTAIN = 2;
 
-  function assumeReceiver(address _receiver) public {
-    // We don't want the receiver to be the Timelock, as that would make our
-    // assertions less meaningful -- most of our tests want to confirm that
-    // proposals can cause tokens to be sent *from* the timelock to somewhere
-    // else. We also can't have the receiver be the zero address because GTC
-    // blocks transfers to the zero address -- see line 546:
-    // https://etherscan.io/address/0xDe30da39c46104798bB5aA3fe8B9e0e1F348163F#code
-    vm.assume(_receiver != TIMELOCK && _receiver != address(0x0));
-  }
-
   function buildProposalData(string memory _signature, bytes memory _calldata)
```

Comment:
Unrelated to this PR, but all these helper methods can be `internal` instead of public